core/shared/src/main/scala/fs2/compression/ZipException.scala

```diff
@@ -19,28 +19,6 @@
  * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
  */
 
-package fs2
-package compression
+package fs2.compression
 
-import java.time.Instant
-
-/** Gunzip decompression results including file properties and
-  * decompressed content stream, used as follows:
-  *   stream
-  *     .through(gunzip[IO]())
-  *     .flatMap { gunzipResult =>
-  *       // Access properties here.
-  *       gunzipResult.content
-  *     }
-  *
-  * @param content Uncompressed content stream.
-  * @param modificationTime Modification time of compressed file.
-  * @param fileName File name.
-  * @param comment File comment.
-  */
-case class GunzipResult[F[_]](
-    content: Stream[F, Byte],
-    modificationTime: Option[Instant] = None,
-    fileName: Option[String] = None,
-    comment: Option[String] = None
-)
+class ZipException(message: String) extends Throwable(message)
```

Comment:
```suggestion
class ZipException(message: String) extends IOException(message)
```

Is this for compatibility with JS? What we've been doing is shim this on the JS side as `fs2.compression.ZipException` and provide type alias on the JVM side.

Here are some examples:
https://github.com/typelevel/fs2/blob/421c2424566caad1cc545a54c6657dc7bd37b00e/io/js/src/main/scala/fs2/io/IOException.scala#L54

https://github.com/typelevel/fs2/blob/421c2424566caad1cc545a54c6657dc7bd37b00e/io/jvm/src/main/scala/fs2/io/ioplatform.scala#L38

Reply:
Aliased `ZipException` to `juz.ZipException` in jvm, new class (extends `IOException`) in js